### ISSUE TITLE
assertions should be in the order expected, actual

### ISSUE THREAD

**Issue by do3cc** [tisto/plonetraining.testing#1]
self.assertEqual('My Talk', talk.title)
I have a book from Kent Beck telling me why, but not in the office. I only found this reference now:
https://github.com/junit-team/junit/wiki/Assertions
Maybe @regebro has memorized the book enough to cite it ;-)

**Comment by tisto** [comment]
Test-Driven Development by Example?

**Comment by tisto** [comment]
You are correct, in TTD by Example the first argument is always the expected one. Couldn't find the explanation right away though...

**Comment by do3cc** [comment]
maybe for the same reason that you should put should put static elements in comparisons on the left hand side of comparisons. But yeah, should bring this book into the office.

**Comment by tisto** [comment]
Nah. Keeping that book under the pillow is also always a good idea. ;)

**Comment by do3cc** [comment]
I am btw. copying your code nearly verbatim into the mastering plone training

**Comment by tisto** [comment]
Not sure if this is the best approach. I plan to continue to work on this and then we will have two diverging sources.

**Comment by tisto** [comment]
Maybe I can move this repo into plone/collective and we can work on this together. I would actually like to keep my approach of testing everything that goes into sphinx.
